src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/Projections/LiveVacancy/LiveVacancy.cs

```diff
@@ -2,10 +2,15 @@
 using System.Collections.Generic;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 
-namespace Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.LiveVacancy
-{
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore.Projections.LiveVacancy
+{
     public class LiveVacancy : QueryProjectionBase
     {
+        public LiveVacancy()
+        {
+            ViewType = QueryViewType.LiveVacancy.TypeName;
```

Comment:
can we enforce this is set by using the base class ctor? looks like it's not always set consistently (eg. dashboard)